Packs/MicrosoftDefenderAdvancedThreatProtection/Scripts/MicrosoftAtpEndpoint/MicrosoftAtpEndpoint.yml

```diff
@@ -0,0 +1,127 @@
+commonfields:
+  id: MicrosoftAtpEndpoint
+  version: -1
+name: MicrosoftAtpEndpoint
+script: ''
+type: python
+tags:
+  - Utilities
+comment: Gets machines that have communicated with Microsoft Defender for Endpoint cloud. At least one of the following arguments is required - IP, hostname, or ID. Otherwise, an error appears.
```

Comment:
Retrieves machines that have communicated with Microsoft Defender for Endpoint cloud. At least one of the following arguments is required: IP, hostname, or ID. Otherwise, an error appears.